### PR TITLE
⚒️ Adding blobbers to createallocation

### PR DESCRIPTION
**Pull request by moldis**
Related to https://github.com/0chain/0chain/issues/668 and https://github.com/0chain/gosdk/issues/297

### CODE REVIEW

File: wasmsdk/sdk.go [hunk 1]
```diff
@@ -1099,36 +1114,36 @@ func CommitToFabric(this js.Value, p []js.Value) interface{} {
 	return promiseConstructor.New(handler)
 }
 
-func GetAllocationMinLock(this js.Value, p []js.Value) interface{} {
```
Comment (moldis) [inline wasmsdk/sdk.go]: @Rizary Why this code removed?

Comment (rizary) [reply wasmsdk/sdk.go]: @cnlangzi has this new feature called `jsbridge` in `wasmsdk/jsbridge`. If you take a look at `wasmsdk/proxy.go` here: https://github.com/0chain/gosdk/pull/301/files#diff-5dca85185f55c99aa4bc09436c42c6721a4c59a9521cd77d067d7d18ae243e15R25 the `GetAllocationMinLock` in `sdk.go` is no longer needed. So we can called directly from `sdk.GetAllocationMinLock`. My commit in this PR is to try jsbridge, yet it's still has a blocker. 

Should we use the normal way first and change it into jsbridge later?

Comment (moldis) [reply wasmsdk/sdk.go]: Got it, make sense

Comment (cnlangzi) [reply wasmsdk/sdk.go]: @Rizary , let us remove duplicated code from sdk.go if possible. thanks.

**Review by cnlangzi — CHANGES_REQUESTED** [review]
please resolve conflicts

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | feature/allocation-with-blobbers  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by cnlangzi — CHANGES_REQUESTED** [review]
1. `CreateAllocation` can be removed from wasm sdk. it is unused anymore.
2. please update zbox to send PreferredBlobbers as `blobbers` input

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | feature/allocation-with-blobbers  |
| zboxcli  | feature/allocation-with-blobbers  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request closed without merging**